overlord/snapstate/snapstate.go

```diff
@@ -68,7 +69,7 @@ func isParallelInstallable(snapsup *SnapSetup) error {
 	return fmt.Errorf("cannot install snap of type %v as %q", snapsup.Type, snapsup.InstanceName())
 }
 
-func doInstall(st *state.State, snapst *SnapState, snapsup *SnapSetup, flags int) (*state.TaskSet, error) {
+func doInstall(st *state.State, fromChange string, snapst *SnapState, snapsup *SnapSetup, flags int) (*state.TaskSet, error) {
```

Comment:
I would prefer fromChange to be the last argument

Reply:
(from IRC: no, actual last)